tox.ini

```diff
@@ -14,8 +15,6 @@ deps =
 commands =
     unit: pytest {posargs}
     flake8: flake8 {posargs}
-    pylint: python setup.py lint
+    pylint: sh -c "git ls-files -co --exclude-standard '*.py' | grep -Ev '^(utils|roles/lib_(openshift|utils)/src)/' | xargs pylint"
```

Comment:
I had a look, pytest-pylint doesn't seem to help with discovery: https://github.com/carsongee/pytest-pylint/blob/master/pytest_pylint.py#L98-L115, it uses the basename just like pylint itself. And it doesn't handle gitignored files.